### DISCUSSION TITLE
How to utilize options of rg (Ripgrep) during grep, such as -t and -F.

### DISCUSSION THREAD

**Discussion by gengdz** [ibhagwan/fzf-lua#1258] [Q&A] [answered]
for example 

rg -tjs 'example' 

rg -tmd -F 'example text'

In this example, there might be alternative approaches, but I am curious to know how I should proceed if I were to use the current options available.

**Comment by ibhagwan** [comment, marked as answer]
If you wanna be able to use these in live grep:
https://github.com/ibhagwan/fzf-lua/wiki#how-can-i-send-custom-flags-to-ripgrep-with-live_grep

For grep, you can always use `grep({ cmd = … })`.

**Reply by gengdz** [reply]
Thanks, it works

I read this document and saw this function before I asked, but the function name `rg_glob_fn` makes me feel that it only works on `--glob` option
